extensions/VisualEditor/wikia/modules/ve/ui/pages/ve.ui.WikiaParameterPage.js

```diff
@@ -16,33 +114,39 @@
  * @param {string} name Unique symbolic name of page
  * @param {Object} [config] Configuration options
  */
-ve.ui.WikiaParameterPage = function VeUiWikiaParameterPage( parameter, name, config ) {
+ve.ui.WikiaParameterPage = function VeUiWikiaParameterPage(parameter, name, config) {
 	var paramName = parameter.getName(),
 		paramType;
 
 	// Parent constructor
-	ve.ui.WikiaParameterPage.super.call( this, parameter, name, config );
+	ve.ui.WikiaParameterPage.super.call(this, parameter, name, config);
 
-	paramType = this.spec.params && this.spec.params[paramName] && this.spec.params[paramName]['type'];
+	paramType = this.spec.params && this.spec.params[paramName] && this.spec.params[paramName].type;
 
 	if (paramType === 'image') {
-		// TODO display something nice
-		this.statusIndicator
-			.setIndicator( 'alert' )
-			.setIndicatorTitle(
-				'This is an image'
-			);
+		this.uploadImageButton = new OO.ui.ButtonWidget({
+			$: this.$,
+			icon: 'image',
+			flags: ['primary'],
+			title: ve.msg('visualeditor-dialog-transclusion-upload-image')
```

Comment:
Yes, this is just a title. And copy is also "Upload image", so I guess this key is appropriate. When we'll be working on image editing, we should set here a condition and check whether it's an upload or image change. "Add image" would be confusing, because images are not added here, only replaced.